src/languages/diff.js

```diff
@@ -16,11 +16,17 @@ export default function(hljs) {
         className: 'meta',
         relevance: 10,
         variants: [
-          {begin: /^@@ +\-\d+,\d+ +\+\d+,\d+ +@@$/},
+          {begin: /^@@ +\-\d+,\d+ +\+\d+,\d+ +@@/},
           {begin: /^\*\*\* +\d+,\d+ +\*\*\*\*$/},
           {begin: /^\-\-\- +\d+,\d+ +\-\-\-\-$/}
         ]
       },
+      {                                                                             
+         className: "meta-string",                                                  
```

Comment:
What is your thinking that this should be meta-string rather than meta or comment?

Reply:
I just tried to get a result that is as close as possible to the `vi` highlighting. But if you prefer `comment` or `meta` I'm fine with it.

![image](https://user-images.githubusercontent.com/1119693/97805388-1335f300-1c56-11eb-80ff-309a2aa45e33.png)

Reply:
My vim doesn't do that... it's all comments... I say lets make it comments for now.  I don't see any good reason it needs to be super emphasized.

Reply:
Strage. vim 8.1 (on Debian) and 8.2 (on Fedora) does...

Reply:
```bash
$ grep "IndexLine" /usr/share/vim/vim82/syntax/diff.vim 
syn match diffIndexLine	"^index \x\x\x\x.*"
hi def link diffIndexLine	PreProc
```

Reply:
I don't doubt you, but just because vim does it doesn't mean we have to do follow.   I took a glance at both ways and just don't think the line deserves such emphasis.